util/scripts/sync_allocation_from_tas.py

```diff
@@ -0,0 +1,158 @@
+from pytas.http import TASClient
+import mysql.connector
+import logging
+import sys
+import argparse
+import configparser
+import os
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+TAS_DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+PORTAL_ALLOCATION_TABLE_NAME = 'allocations_allocation'
+PORTAL_AUTH_USER_TABLE_NAME = 'auth_user'
+
+TAS_TO_PORTAL_MAP = {'project': 'project_charge_code',
```

Comment:
Does it make sense to `import` this somehow? Then we don't have to worry about it being out of sync. Same applies for TAS_DATE_FORMAT

Reply:
I don't think so...